app/src/main/java/edu/uc/groupproject/covid19tracker/ui/main/ItemAdapter.kt

```diff
@@ -21,20 +23,21 @@ class ItemAdapter(var context: Context, var view_layout: Int, var arrayCaseList:
         return arrayCaseList.size
     }
 
+    @SuppressLint("SetTextI18n", "ViewHolder")
```

Comment:
Added SuppressLint to suppress any warnings, I believe it should help with speed: https://developer.android.com/reference/android/annotation/SuppressLint